Code/Framework/AzToolsFramework/AzToolsFramework/UI/Outliner/EntityOutlinerWidget.cpp

```diff
@@ -905,7 +905,7 @@ namespace AzToolsFramework
 
     void EntityOutlinerWidget::SetDefaultTreeViewEditTriggers()
     {
-        m_gui->m_objectTree->setEditTriggers(QAbstractItemView::SelectedClicked | QAbstractItemView::EditKeyPressed);
+        m_gui->m_objectTree->setEditTriggers(QAbstractItemView::EditKeyPressed);
```

Comment:
Is this going to break the functionality of clicking on a selected item to rename?